### ISSUE TITLE
Import Repository from SGit without copying

### ISSUE THREAD

**Issue by Tcc100** [maks/MGit#107]
When I migrated from SGit I tried to import my local repository, but MGit tried to copy the repository to the path where it already was and the import failed.
As a workaround I was able to rename the folder, import to the previous location and delete the renamed folder.
MGit might also automatically show or import on demand all git repositories that are stored in the root storage location.

**Comment by maks** [comment]
Sorry this should be documented better, but for this use case, you need to check the "Import as external repo" checkbox in the dialog displayed after you choose a folder to import as a repo.
This will import the repo "in place" rather than trying to copy it to a new folder in the default root folder for repos.

**Comment by Tcc100** [comment]
Ok I see, my confusion arised from the german translation which states "as remote repository" which has a different meaning in the context of git.
If I import my repository as external repository it is titled "... (external)" and the behavior of the deletion is changed, neither way I can decide if I want the files to be deleted or not.
In the context of migration I'd prefer to import my repository as local one.

**Comment by maks** [comment]
Yes, that is a bug, sorry about that. When importing a repo that's already within the folder set as the root for repos, it should not be marked as "external" so I'll track fixing that with this issue.
